pkgs/applications/audio/spotify/update.sh

```diff
@@ -38,13 +38,14 @@ snap_info=($(
 # "revision" is the actual version identifier on snapcraft, the "version" is
 # just for human consumption. Revision is just an integer that gets increased
 # by one every (stable or unstable) release.
+
 revision="${snap_info[0]}"
-sha512="${snap_info[1]}"
+# We need to escape the slashes
+hash="$(nix hash to-sri --type sha512 ${snap_info[1]} | sed 's/\//\\\//g')"
```

Comment:
```
$ nix hash
error: experimental Nix feature 'nix-command' is disabled; use '--extra-experimental-features nix-command' to override
```

Please don't make nixpkgs rely on experimental Nix features.

Reply:
how should I convert the hash then? 
The package is already using `hash` and we can't use the `sha512` hash from `snap_info` without converting it to SRI

Reply:
Doesn't nix-hash work? (If `hash` is incompatible with (non-experimental) Nix, then I don't think we should move nixpkgs to `hash` yet.)

Reply:
Ok it works with slightly modified command, I always used `nix hash` instead of `nix-hash` so I assumed `nix hash` was same as `nix-hash`, fixing it